src/components/Topbar/Search.js

```diff
@@ -1,17 +1,35 @@
+import React from 'react';
 import styled from 'styled-components';
 
 import { baselineBreakpoint } from '../../config';
 
 // Temporary placeholder for search to align the Topbar items nicely
-const Search = styled.div`
+const Wrapper = styled.div`
   width: 60px;
   height: 60px;
 
   @media (min-width: ${baselineBreakpoint}px) {
-    // Same width as the logo in the left for symmetry
-    width: 120px;
-    height: 72px;
+    width: 220px;
+    height: 40px;
+    border: 1px solid ${props => props.theme.searchBorderColor};
+    border-radius: 20px;
```

Comment:
Remove the border from here and and add it directly to input.

Input could contain something like:
```css
  width: 220px;
  height: 40px;
  border: 1px solid #d1d1d1;
  border-radius: 20px;
  padding: 0 10px 0 20px;
  font-size: 16px;
```

Reply:
To the **src/components/GlobalStyles.js**, you could add something like this:
```css
  .algolia-autocomplete {
    ${fonts['CircularStd-Book'].styles}

    & .algolia-docsearch-suggestion--title {
      ${fonts['CircularStd-Bold'].styles}
    }
  }
```

After button style-rules.

Reply:
> => i.e remove the border from here. You can add it directly to Input.

How about the magnifying glass, should it be outside the input, so that there would be a container element that holds both?

Reply:
Probably the icon can also go into the input, having the border in the input instead of a wrapping `div` does feel more natural indeed.